tools/make-clean.sh

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env bash
+#!/usr/bin/env bash -e
 #
 # This script assumes a linux environment
 
 echo "*** uBlock: Cleaning."
-rm -R dist/build
+rm -Rf dist/build
```

Comment:
Note that sometimes a script can "fail" when it has not really failed at all. In this case, the `rm` command exits with a non-zero code if `dist/build` does not exist (e.g. `make-clean.sh` is run twice). The `-f` option ensures that this is not the case.